typography/main.scss

```diff
@@ -1,19 +1,15 @@
 @include nUiCriticalStart('typography');
 
-$_n-ui-typography-applied: false !default;
-$_n-ui-typography-is-silent: nUiHas('typography') == false or $_n-ui-typography-applied == true;
-
 $o-fonts-path: 'https://next-geebee.ft.com/build/v2/files/o-fonts-assets@1.3.0/';
 // TODO - once o-typography lets us control font loading in non-silent mode we should unsilence it
-//$o-typography-is-silent: $_n-ui-typography-is-silent;
```

Comment:
I don't know I'm afraid, but I think there was some discussion to be had about responsive fonts too as the class names exposed by o-typography are currently of little use to us.